examples/conn/plot_fit.py

```diff
@@ -0,0 +1,55 @@
+"""
+FIT: Feature specific information transfer
+==================================================================
+
+This example illustrates how to discover a feature (e.g. sensory stimuli) in information transfer between brain regions.
```

Comment:
This line is probably too long and might not render correctly in the gallery of examples. Can you make it 79 characters long? Same for the `==================================================================`, you can stop it under the final letter "r" of *transfer*